src/Tank/Tank.hpp

```diff
@@ -45,16 +48,19 @@ class Tank {
   [[nodiscard]] sf::Vector2f get_position() const;
   [[nodiscard]] sf::FloatRect get_body_rect() const;
   [[nodiscard]] float get_current_speed() const;
+  [[nodiscard]] bool is_alive() const;
 
  private:
   void draw_tracks(sf::RenderWindow& window);
   void update_position();
 
   sf::Vector2f mPos;
   float mCurrentSpeed = 0.0f;
+  int mHealth;
```

Comment:
Add default value 100

Reply:
Here I run into some trouble:

Adding default value in in-class initialization would mean that I should remove the `health` parameter from the c-tor right (at least the default value for it since it will always override the in-class default value)?

Now that means I also have to add in-class initialization to `mHealthBar` (otherwise I can't initialize it in c-tor initializer list since the `mHealth` doesn't have a value yet), so I opted to this and I added the `TANK_HEALTH` const.

Reply:
Sounds good! 
Of course constructor can still be the same - adding default value in class member only prevents us for future mistakes.